python_modules/dagster/dagster/_daemon/sensor.py

```diff
@@ -65,6 +66,13 @@
 
 MIN_INTERVAL_LOOP_TIME = 5
 
+# When retrying a tick, how long to wait before ignoring it and moving on to the next one
+# (To account for the rare case where the daemon is down for a long time, starts back up, and
+# there's an old in-progress tick left to finish that may no longer be correct to finish)
+MAX_TIME_TO_RESUME_TICK_SECONDS = 60 * 60 * 24
+
+MAX_FAILURE_RETRIES = 3
```

Comment:
i think we should default this to 0 not 3, because (unlike AMP ticks) sensor ticks can fail for all kinds of reasons, and it might be surprising or undesirable to start automatically retrying them (particularly if the sensor function has side effects). Schedule retries work that way currently.

That value is currently instance config on the scheduler - recognizing that we are wildly inconsistent across different daemons here it might be good to keep the same instance config behavior for sensors too

Reply:
just to bring the offline discussion back here -- the thing that this constant is doing is fairly different from what the Schedule retry thing is doing. I've changed the name (open to further changes there) and knocked the value down to 1 (although maybe 3 is fine there, pretty indifferent). Do those changes feel sufficient to you?